pkg/discovery/queries.go

```diff
@@ -260,7 +261,8 @@ func filterResources(gvrs map[schema.GroupVersion][]metav1.APIResource, namespac
 }
 
 // QueryPodLogs gets the pod logs for each pod in the given namespace.
-func QueryPodLogs(kubeClient kubernetes.Interface, recorder *QueryRecorder, ns string, cfg *config.Config) error {
+func QueryPodLogs(kubeClient kubernetes.Interface, recorder *QueryRecorder, ns string, cfg *config.Config,
+	podLogOptions *v1.PodLogOptions) error {
```

Comment:
Lets not add this new parameter; they can get it just from the cfg so it doesnt add any new info.